tests/.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
Add new line at end of file, actually, just copy the one from the other services can le